src/main/java/vic/cinema/Main.java

```diff
@@ -76,8 +77,13 @@ public static void main(String[] args) throws AuthenticationException {
         shoppingCartService.addSession(movieSession, user2);
         System.out.println("Cart with tickets: ");
         System.out.println(shoppingCartService.getByUser(user2));
+
+        System.out.println("---- Order ----");
+        OrderService orderService =
+                (OrderService) injector.getInstance(OrderService.class);
+        orderService.completeOrder(shoppingCartService.getByUser(user2).getTickets(), user2);
         shoppingCartService.clear(shoppingCartService.getByUser(user2));
```

Comment:
this should be done inside `completeOrder()`

Reply:
Done